cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,43 @@
 /// <reference types='cypress' />
+import faker from 'faker';
```

Comment:
Please use in this task pageObject model) you should import homePage and other pages. Use provided functions in this classes in your flow